isbnbot/normalize_isbns.py

```diff
@@ -14,6 +14,9 @@
 from os import makedirs
 
 
+ALLOWED_ISBN_CHARS = {'0', '1', '2','3', '4', '5', '6', '7', '8', '9', 'X', '-'}
```

Comment:
Do we want lowercase x here?

```suggestion
ALLOWED_ISBN_CHARS = {'0', '1', '2','3', '4', '5', '6', '7', '8', '9', 'X', 'x', '-'}
```